cnf-tests/testsuites/e2esuite/rps/rps.go

```diff
@@ -0,0 +1,268 @@
+package rps
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	apitypes "k8s.io/apimachinery/pkg/types"
+	"k8s.io/klog"
+	"k8s.io/utils/cpuset"
+
+	netattdefv1 "github.com/k8snetworkplumbingwg/network-attachment-definition-client/pkg/apis/k8s.cni.cncf.io/v1"
+	sriovClean "github.com/k8snetworkplumbingwg/sriov-network-operator/test/util/clean"
+	sriovtestclient "github.com/k8snetworkplumbingwg/sriov-network-operator/test/util/client"
+	sriovcluster "github.com/k8snetworkplumbingwg/sriov-network-operator/test/util/cluster"
+	sriovnamespaces "github.com/k8snetworkplumbingwg/sriov-network-operator/test/util/namespaces"
+	"github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/pkg/client"
+	"github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/pkg/discovery"
+	"github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/pkg/namespaces"
+	"github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/pkg/networks"
+	"github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/pkg/nodes"
+	"github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/pkg/performanceprofile"
+	performancev2 "github.com/openshift/cluster-node-tuning-operator/pkg/apis/performanceprofile/v2"
+	"github.com/openshift/cluster-node-tuning-operator/pkg/performanceprofile/controller/performanceprofile/components"
+	profileutil "github.com/openshift/cluster-node-tuning-operator/pkg/performanceprofile/controller/performanceprofile/components/profile"
+	ntonodes "github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils/nodes"
+)
+
+var (
+	sriovclient                    *sriovtestclient.ClientSet
+	profile                        *performancev2.PerformanceProfile
+	machineConfigPoolName          string
+	performanceProfileName         string
+	enforcedPerformanceProfileName string
+	dpdkResourceName               = "dpdknic"
+)
+
+const sriovNetworkName = "test-sriov-with-rps-configuration"
+
+func init() {
+	sriovclient = sriovtestclient.New("")
+	performanceProfileName = os.Getenv("PERF_TEST_PROFILE")
+	if performanceProfileName == "" {
+		performanceProfileName = "performance"
+	} else {
+		enforcedPerformanceProfileName = performanceProfileName
+	}
+}
+
+var _ = Describe("[rps] RPS configuration for SR-IOV devices", Ordered, func() {
```

Comment:
I think in the name you need to add also sriov so when we do a focus we will find this test no?